HRIS.Services/Services/ChartService.cs

```diff
@@ -97,16 +97,6 @@ public async Task<List<ChartDto>> GetAllCharts()
 
             var labels = roleDictionaries.Values.SelectMany(dict => dict.Keys).Distinct().OrderBy(label => label).ToList();
             
-            foreach (var dictionary in roleDictionaries.Values)
```

Comment:
Why did you take this out?

Reply:
I am not sure. I did not work on charts, I think this is some weird merge problem...